frame/staking/src/lib.rs

```diff
@@ -2368,6 +2380,76 @@ impl<T: Config> Module<T> {
 		<SnapshotNominators<T>>::kill();
 	}
 
+	/// Modify the target validator count based on the staking participation.
+	fn dynamic_damping_validator_count(
+		exposures: Vec<BalanceOf<T>>
```

Comment:
name is kinda misleading, no? You'd expect `exposures` to be `Vec<Exposure<_>>`, but it is not. I'd just call this `stakes` or `exposures_totals` or anythign else.